pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -283,9 +288,24 @@ func deploymentOnPVC(c *Cluster, osd OSDInfo, pvcName string, config *provisionC
 		return nil, errors.Wrapf(err, "failed to generate deployment for %s", osdLongName)
 	}
 
+	err = setOSDProperties(c, osdProps, osd)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to prepare for deployment %s", osdLongName)
```

Comment:
```suggestion
		return nil, errors.Wrapf(err, "failed to prepare deployment for %s", osdLongName)
```
and there is already a comment about using  `%q`

Reply:
@sp98 You are correct!